website/docs/cli.md

```diff
@@ -666,4 +666,91 @@ corresponding to the library release version is used:
 
 ```shell
 upgrade table
+```
+
+### Change Hudi Table Type
+There are cases we want to change the hudi table type. For example, change COW table to MOR for more efficient and 
+lower latency writing; change MOR to COW for for better read performance or something else.
```

Comment:
lower latency writing; change MOR to COW for for better read performance or something else. ->

lower latency ingestion; change MOR to COW for for better read performance and compatibility with downstream engines.